### DISCUSSION TITLE
Possible to detect good P8 hybrid without both original sources?

### DISCUSSION THREAD

**Discussion by nateify** [R3S3t9999/DoVi_Scripts#6] [Q&A] [answered]
In the Youtube tutorial for detecting if WEB DV can be properly injected into a BD source, you had both the BD and WEB-DL sources separately.

If I for example download a Framestor P8 hybrid after the fact, and it is not on the Spreadsheet, is it possible to use only that 1 source to tell if the P8 hybrid was correct? I noticed for example "RIPD 2013" has "different grade + MDL mismatch" and lists Framestor as the source so I wonder what other Framestor releases might be "incorrect".

**Comment by R3S3t9999** [comment, marked as answer]
Yes, you can measure HDR10 with madVR (workflow 6-2) and [compare](https://slow.pics/c/ZARC6Q0c) with the DV L1 plot.

**Reply by nateify** [reply]
I ran the 6-2 workflow on this particular BD/P8 hybrid but I am not sure why my result differs and shows MaxCLL of 10k nits compared to the existing HDR10 plot which was found in the Google Drive folder https://slow.pics/c/3wrzndcO

**Comment by R3S3t9999** [comment]
Check your[ lavfilters settings](https://ibb.co/c28Qtds).

**Reply by nateify** [reply]
Thanks, enabling the hardware decoder did the trick.
